independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json

```diff
@@ -0,0 +1,514 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "De Lijn Zoek Open Data Services (v1)",
+    "description": "Get locations of public bus stations and extra info from the Flemish bus company 'De Lijn'",
+    "version": "v1"
+  },
+  "host": "api.delijn.be",
+  "basePath": "/DLZoekOpenData/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/zoek/haltes/{zoekArgument}": {
+      "get": {
+        "summary": "zoekHaltes",
```

Comment:
@Lschockaert all operations summaries should be a valid sentence. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please change this for all the operations summaries in the swagger file.